Internal Server Error: /core/competition_schedule/MotoGP/
Traceback (most recent call last):
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/core/views.py", line 63, in competition_schedule
    return render(request, 'core/competition_schedule.html', {
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 563, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 26: 'static', expected 'endblock'. Did you forget to register or load this tag?
"GET /core/competition_schedule/MotoGP/ HTTP/1.1" 500 184938
Internal Server Error: /core/competition_schedule/MotoGP/
Traceback (most recent call last):
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/core/views.py", line 63, in competition_schedule
    return render(request, 'core/competition_schedule.html', {
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 563, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 26: 'static', expected 'endblock'. Did you forget to register or load this tag?
Internal Server Error: /core/competition_schedule/Formula 1/
Traceback (most recent call last):
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/core/views.py", line 63, in competition_schedule
    return render(request, 'core/competition_schedule.html', {
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/dom974/Development/Project/Project-Sports-Replay-Forum/.venv/lib/python3.12/site-packages/django/template/base.py", line 563, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 26: 'static', expected 'endblock'. Did you forget to register or load this tag?
"GET /core/competition_schedule/Formula%201/ HTTP/1.1" 500 186261
